packages/tracing/src/idletransaction.ts

```diff
@@ -189,15 +206,35 @@ export class IdleTransaction extends Transaction {
     this.spanRecorder.add(this);
   }
 
+  /**
+   * Creates an idletimeout
+   */
+  private _cancelIdleTimeout(): void {
+    if (this._idleTimeoutID) {
+      global.clearTimeout(this._idleTimeoutID);
+      this._idleTimeoutID = undefined;
+    }
+  }
+
+  /**
+   * Creates an idletimeout
+   */
+  private _startIdleTimeout(end?: Parameters<IdleTransaction['finish']>[0]): void {
+    this._cancelIdleTimeout();
```

Comment:
I wonder what test covers the behaviour changes?

Does its 2 test cases will to it?
https://github.com/getsentry/sentry-javascript/pull/4442/files#diff-69402bcbb230fa3d9774ae3cc7f75cecc78e369775e6fa222ae406f1d4dcfef2R197-R230